charts/datadog/values.yaml

```diff
@@ -530,6 +530,10 @@ datadog:
       # datadog.apm.instrumentation.libVersions -- Inject specific version of tracing libraries with Single Step Instrumentation (preview).
       libVersions: {}
 
+      # datadog.apm.instrumentation.targets -- Enable target based workload selection (preview).
+      # Requires Cluster Agent 7.64.0+
+      targets: []
+
```

Comment:
Most of complex properties include commented out comprehensive example which serves a documentation and which can be copied or uncommented for testing, e.g. 
https://github.com/DataDog/helm-charts/blob/4f0be9666212069cd55bf41574b3c0e2c80b1ea5/charts/datadog/values.yaml#L664-L675

Could you please provide one for this?

Reply:
Updated!